apps/drupal/particle.layouts.yml

```diff
@@ -0,0 +1,56 @@
+particle_one_col:
+  label: 'Particle one column'
+  path: layout/particle-onecol
+  template: particle-onecol
+  category: 'Columns: 1'
```

Comment:
Is `category` something we could leverage? ie a "Particle" category?

Reply:
@illepic --yes, I believe so! To confirm, do you mean replace "Columns: 1" with "Particle"?